cluster-config/staging/member-base/namespaces/kustomization.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - group-sync-operator
-  - pipelines-as-code
-  - build-templates
-  - build-service
-  - tekton-results
-  - gitops
-  - application-service
+- group-sync-operator
+- pipelines-as-code
+- build-templates
+- build-service
+- tekton-results
+- gitops
+- appstudio-workload-monitoring
```

Comment:
too many changes in here, can you please keep the current indentation? in addition, it seems like you removed the `application-service` namespace.